### DISCUSSION TITLE
Dark mode

### DISCUSSION THREAD

**Discussion by lucassimon** [Lednerb/bilberry-hugo-theme#526] [Ideas]
It would be great if there was a button to change the theme to dark

**Comment by igor-baiborodine** [comment]
@lucassimon, yes, having a dark theme variation would be nice, but currently, it's not a priority. I'm presently concentrated on [`v4`](https://github.com/Lednerb/bilberry-hugo-theme/discussions/523) development, and I don't see that it will be implemented shortly unless someone volunteers for this task.

Duplicate of https://github.com/Lednerb/bilberry-hugo-theme/discussions/315.

I'm converting this issue to a Discussions/Ideas item.
